f/assignGear/f_assignGear_fia.sqf

```diff
@@ -188,8 +188,8 @@ _bag = "B_AssaultPack_cbr";
 _bagLarge = "B_Kitbag_cbr";
 _bagdiver =  "B_AssaultPack_blk";		// used by divers
 _baguav = "B_UAV_01_backpack_F";			// used by UAV operator
-_baghmgg = "B_HMG_01_weapon_F";			// used by Heavy MG gunner
-_baghmgag = "B_HMG_01_support_F";			// used by Heavy MG assistant gunner
+_baghmgg = "I_G_HMG_02_weapon_F";			// used by Heavy MG gunner
```

Comment:
There's B_G and O_G variants of the M2 for FIA and NPR. Probably doesn't matter though, and I think they're slightly sketchy at the moment (they work but the tripods are mislabeled as Syndikat).

Reply:
All the bags look exactly the same, and B_G and O_G would both be also labelled as FIA. I just used the one visible in the editor arsenal - I can't think of any aspect in which it would make any difference.